cpp/src/arrow/compute/kernels/aggregate_test.cc

```diff
@@ -2543,7 +2543,9 @@ void CheckModes(const Datum& array, const ModeOptions options,
   for (int i = 0; i < out_array.length(); ++i) {
```

Comment:
Could perhaps also add `ARROW_SCOPED_TRACE("Mode options = ", options.ToString())` somewhere above to provide better insight into what is computed?

Reply:
Done